src/sentry/static/sentry/app/views/performance/vitalDetail/vitalPercents.tsx

```diff
@@ -38,7 +38,10 @@ export default function VitalPercents(props: Props) {
                       })
                 }
               >
-                <IconFire color={vitalStateColors[p.vitalState] as Color} />
+                <IconFire
+                  style={{verticalAlign: 'middle'}}
+                  color={vitalStateColors[p.vitalState] as Color}
```

Comment:
Should this be cast as `Color` where we export `vitalStateColors`?

Reply:
Possibly. I'll look into it.

Reply:
We can't just change `vitalStateColors` to be `Color`s, because it's also used at:

https://github.com/getsentry/sentry/blob/f2e173d1a93b1da03b819c3cb02e437a4575c8ca/src/sentry/static/sentry/app/views/performance/vitalDetail/table.tsx#L430

I think this is fine for now, and if we want to change vitalStateColors that can be done in a separate PR.